valid_parentheses.py

```diff
@@ -0,0 +1,38 @@
+""" Given a string s containing just the characters
+'(', ')', '{', '}', '['and ']',
+determine if the input string is valid.
+
+An input string is valid if:
+
+Open brackets must be closed by the same type of brackets.
+Open brackets must be closed in the correct order. """
+
+
+def isValid(strings: str) -> bool:
+    output = []
+    braces = {
+        ")": "(",
+        "]": "[",
+        "}": "{"
+    }
+    for symbols in strings:
```

Comment:
nit: `char` instead of `symbols`? specifically for looping through string, also it shouldn't be plural?